docs/cody/usage-and-pricing.mdx

```diff
@@ -47,6 +56,28 @@ To revert back to Cody Free from Pro:
 
 To upgrade from Cody Pro to Cody Enterprise, you should [Contact Sales](https://sourcegraph.com/contact/request-info) and connect with one of our account teams. They will help you set up your account and start with Cody Enterprise.
 
+## Billing FAQs for Cody Pro
+
+### What is Cody's pricing plan?
+
+You can get this info from our [pricing page](https://sourcegraph.com/pricing?product=cody).
+
+### When are payments taken?
+
+We charge payments at the beginning of each billing cycle, and they get automatically renewed once you've added your credit card details.
+
+### What is a billing cycle?
+
+The billing cycle refers to the start date and depends on your account creation date. You can view your current and previous billing cycles from the **My subscription** tab.
+
+### What payment methods are available?
+
+We only support payments through a credit card.
+
+### What happens when I cannot pay or don't want to renew the Pro plan?
+
+If you are on a Pro plan trial, you must add a credit card to your account to continue on a Pro plan. If you do not want to renew Cody Pro, you will automatically downgrade to Cody Free.
+
```

Comment:
We should have a section about Refunds. Something like: 

- We don't offer refunds, but if you have a problem with your Cody Pro plan, please write to [support@sourcegraph.com](mailto:support@sourcegraph.com) and we can work with you to resolve your problem.

Reply:
CC: @enriquegh

Reply:
Agreed. The blurb Kevin added looks good